ertviz/models/response.py

```diff
@@ -4,32 +4,38 @@
 
 
 class Response:
-    def __init__(self, ref_url):
-        schema = get_schema(api_url=ref_url)
-        self.name = schema["name"]
-        self.ensemble_id = schema["ensemble_id"]
-        self._axis_url = schema["axis"]["data_url"]
+    def __init__(self, name, ref_url):
+        self._schema = None  # get_schema(api_url=ref_url)
+        self._ref_url = ref_url
+        self.name = name
         self._axis = None
-        self._data_url = schema["alldata_url"]
         self._data = None
         self._realizations = []
         self._observations = []
-        if "realizations" in schema:
-            self._realizations_schema = schema["realizations"]
-            self._realizations = None
-        if "observations" in schema:
-            self._observations_schema = schema["observations"]
-            self._observations = None
+        self._realizations = None
+        self._observations = None
+
+    @property
+    def ensemble_id(self):
+        if not self._schema:
+            self._schema = get_schema(api_url=self._ref_url)
+        return self._schema["ensemble_id"]
 
     @property
     def axis(self):
+        if not self._schema:
```

Comment:
this's quite repetitive, could we make a function e.g. `_update_schema` 
```python
def _update_schema
   if not self._schema:
          self._schema = get_schema(api_url=self._ref_url)
```

Reply:
:+1: